.travis.yml

```diff
@@ -1,5 +1,7 @@
 # https://travis-ci.org
 
+sudo: required
```

Comment:
Maybe that was part of the whitelisting we got earlier, because it was before/around when they added the containers.

I suppose this isn't an issue, though.

-[Unknown]